src/StreamJsonRpc/JsonMessageFormatter.cs

```diff
@@ -25,13 +26,18 @@ namespace StreamJsonRpc
     /// <summary>
     /// Uses Newtonsoft.Json serialization to serialize <see cref="JsonRpcMessage"/> as JSON (text).
     /// </summary>
-    public class JsonMessageFormatter : IJsonRpcMessageTextFormatter
+    public class JsonMessageFormatter : IJsonRpcMessageTextFormatter, IJsonRpcInstanceContainer
```

Comment:
Add remarks here:

```cs
/// <remarks>
/// Each instance of this class may only be used with a single <see cref="JsonRpc" /> instance.
/// </remarks>
```

Reply:
Oops. You've got `/// ///` on the first line of remarks.

---
In reply to: [304600538](https://github.com/milopezc/vs-streamjsonrpc/pull/1#discussion_r304600538) [](ancestors = 304600538)